rollup-http/rollup-http-client/src/rollup.rs

```diff
@@ -20,6 +20,8 @@ use std::fmt;
 
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub struct AdvanceMetadata {
+    pub chain_id: u64,
+    pub app_contract: String,
```

Comment:
Silly question. These `String` here are always hex-encoded, right? Otherwise, wouldn't Rust complain when the data isn't a proper UTF-8 string?

Reply:
Should there be a `prev_randao` here?